shared/chat/conversation/input-area/normal/mention-input.js

```diff
@@ -34,6 +36,10 @@ class MentionInput extends React.Component<MentionInputProps, MentionState> {
     this._inputRef = input
   }
 
+  _setLastSelectionKey = (lastSelectionKey: string) => {
+    this.setState(p => (p.lastSelectionKey !== lastSelectionKey ? {lastSelectionKey} : null))
```

Comment:
so the `lastSelectionKey` doesn't need to be part of the state, since the state is only for things that affect rendering. So it could be another instance variable. Also, we only really care about Tab vs. Enter, so I'd suggest:

```
_lastPickSelectionKey: 'Enter' | 'Tab'
```

and have `triggerPickSelectionCounter` take a `key: 'Enter' | 'Tab'` parameter, and set `_lastPickSelectionKey` based on that.